agent-c4/build.gradle

```diff
@@ -39,7 +39,7 @@ shadowJar {
     }
 }
 
-jar.enabled = false
+jar.enabled = true
```

Comment:
This is necessary to solve:
```
19:24:38.807 ERROR Exception in thread "main" java.lang.NoClassDefFoundError: com/datastax/oss/cdc/agent/Mutation
    19:24:38.807 ERROR 	at com.datastax.oss.cdc.backfill.factory.BackfillFactory.newPulsarImporter(BackfillFactory.java:37)
```

exception in the CLI. It is enabled on the dse4-agent (so this is how it was working before) but now I'm switching to the c4 agent implementation to avoid the dependency on dse-db so external users can build